man/docker-inspect.1.md

```diff
@@ -24,14 +25,29 @@ each result.
 **-f**, **--format**=""
     Format the output using the given go template.
 
+**--type**=*container*|*image*
+    Return JSON for specified type, permissible values are "image" or "container"
+
 # EXAMPLES
 
+Getting information on an image where image name conflict with the container name.
+e,g both image and container are named rhel7.
```

Comment:
Really minor; some dots/comma's are incorrect here. *But I'm okay with leaving that for someone else to fix.*

Think it should be;
```Markdown
Getting information on an image where image name conflict with the container name, 
e.g., both image and container are named rhel7.
```